lib/streamlit/navigation/page.py

```diff
@@ -141,6 +153,10 @@ def title(self) -> str:
     def icon(self) -> str:
         return self._icon
 
+    @property
+    def url_path(self) -> str:
+        return self._url_path
```

Comment:
Only one conversation might be to change this to the following:

`return "" if self._default else self._url_path`